app/views/posts/_form.html.erb~

```diff
@@ -0,0 +1,22 @@
+<%= form_with(model: post) do |f| %>
```

Comment:
Same comment as not showing up as a valid file due to `~` end of the file extension.